contracts/core/EntryPoint.sol

```diff
@@ -261,7 +263,12 @@ contract EntryPoint is IEntryPoint, StakeManager, NonceManager, ReentrancyGuard,
         unchecked {
             // handleOps was called with gas limit too low. abort entire bundle.
             if (
-                gasleft() < callGasLimit + mUserOp.verificationGasLimit + 5000
+                gasleft() <
+                callGasLimit +
+                mUserOp.verificationGasLimit +
+                mUserOp.paymasterVerificationGasLimit +
```

Comment:
why do you need pmVGL inside inner ?
afaik, you only need postGL

Reply:
Ok